lib/ecto/uuid.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Ecto.UUID do
+  @moduledoc """
+  An Ecto type for UUIDs.
+  """
+
+  @behaviour Ecto.Type
+
+  @doc """
+  The Ecto primitive type.
+  """
+  def type, do: :uuid
+
+  @doc """
+  UUIDs are never blank.
+  """
+  def blank?(_), do: false
```

Comment:
That was the way I did it in Crudex, but then I thought, an empty binary is actually no valid UUID so casting should fail in my opinion.